### DISCUSSION TITLE
Can I train using PNG semantic masks as labels instead of COCO?

### DISCUSSION THREAD

**Discussion by fschvart** [facebookresearch/detectron2#4893] [General]
I tried searching and couldn't an answer, is it possible to train Detectron2-based models with semantic mask annotations? 
I have complex masks that I wasn't able to convert to COCO polygon annotations, any help in training with .PNG labels would be greatly appreciated

**Comment by SouLeo** [comment]
Have you found any further information on this? I'm in a similar situation.
